build.sbt

```diff
@@ -4,17 +4,17 @@ lazy val buildSettings = Seq(
   organization := "com.github.finagle",
   version := "0.11.0-M4",
   scalaVersion := "2.11.8",
-  crossScalaVersions := Seq("2.10.6", "2.11.8")
+  crossScalaVersions := Seq("2.11.8", "2.12.0")
```

Comment:
```crossScalaVersions``` defines scala versions for which finch itself is built and published, so it works fine